Dockerfile

```diff
@@ -11,12 +11,14 @@ WORKDIR /usr/src/app
 # rebuild image when package.json or lock has changed
 COPY package*.json ./
 
-# install dependencies
-RUN npm ci --only=production
```

Comment:
Why did you move this? It will invalidate all layers after every project change. We only want to invalidate the layers when the package*.json has changed.

Reply:
I did not remove it, it's run with the same command as `rm -fr node_modules` to avoid creating an extra layer. I changed `--only=production` to `--omit=dev` because the former is deprecated.